### PR TITLE
publish flutter CI

### PR DESCRIPTION
**Pull request by roeierez**

### CODE REVIEW

File: .github/workflows/publish-flutter.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ inputs.ref || github.sha }}
+          path: breez-sdk
+
+      - uses: actions/download-artifact@v3
+        with:
+          name: android-jniLibs
+          path: flutter/sdk-flutter/android/src/main
+
```
Comment (JssDWt) [inline .github/workflows/publish-flutter.yml:45]: Is this missing ios artifacts?

File: .github/workflows/publish-flutter.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        working-directory: flutter
+        run: |
+          git config --global user.email github-actions@github.com
+          git config --global user.name github-
+          rm -f ../breez-sdk/libs/sdk-flutter/ios/breez_sdk.podspec.dev
+          mv ../breez-sdk/libs/sdk-flutter/ios/breez_sdk.podspec.production ../breez-sdk/libs/sdk-flutter/ios/breez_sdk.podspec
+          cp -r ../breez-sdk/libs/sdk-flutter/ios .
+          cp -r ../breez-sdk/libs/sdk-flutter/android .
+          cp -r ../breez-sdk/libs/sdk-flutter/lib .
+          cp ../breez-sdk/libs/sdk-flutter/pubspec.yaml .
```
Comment (JssDWt) [inline .github/workflows/publish-flutter.yml:56]: The pubspec.yaml probably needs a version bump?

**Review by JssDWt — COMMENTED** [review]
What is the purpose of the `breez-sdk-flutter` repo? Is it a clone of libs/sdk-flutter with binaries included? If that's the only reason we need it, we could also make CI put the binaries in the right place inside the breez-sdk repo, and then publish to pub.dev.

File: .github/workflows/publish-flutter.yml (hunk 1 above)
Comment (roeierez) [reply .github/workflows/publish-flutter.yml:45]: No, iOS is not pushed to this repo and the dependency is reflected via podfile.

**Comment by roeierez** [comment]
> What is the purpose of the `breez-sdk-flutter` repo? Is it a clone of libs/sdk-flutter with binaries included? If that's the only reason we need it, we could also make CI put the binaries in the right place inside the breez-sdk repo, and then publish to pub.dev.

breez-sdk-flutter is the flutter plugin published to git.Currently we don't have a published version to pub.dev.
The next step is indeed to publish to pub.dev and then if we figure out that we don't need the git repo for the plugin basis then I agree we can change the CI to use the folder inside breez-sdk directly.

File: .github/workflows/publish-flutter.yml (hunk 2 above)
Comment (roeierez) [reply .github/workflows/publish-flutter.yml:62]: Good point!

Comment (roeierez) [reply .github/workflows/publish-flutter.yml:59]: Fixed

File: .github/workflows/publish-flutter.yml [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+          cp ../breez-sdk/libs/sdk-flutter/pubspec.lock .          
+           
+      - name: Tag the Flutter package        
+        working-directory: flutter
```
Comment (JssDWt) [inline .github/workflows/publish-flutter.yml:63]:
```suggestion
        working-directory: flutter
        if: ${{ inputs.publish }}
```

**Review by JssDWt — COMMENTED** [review]
Last nit then, I think it's good!

**Review by JssDWt — APPROVED** [review]

**Pull request merged**